packages/rancher-vsphere-csi/package.yaml

```diff
@@ -1,4 +1,4 @@
 url: https://github.com/rancher/vsphere-charts.git
 subdirectory: charts/rancher-vsphere-csi
-commit: 0a91c2fe20a9ac83512ac1b06ed712b30653b849
-packageVersion: 00
+commit: 6f2bb5eb0de34a21cfbc0232973217d1dda1fe69
+packageVersion: 01
```

Comment:
this can stay at 0, since the upstream chart version pointed to by this commit has been bumped